tests/modules/lib/cddl-gen/decode/CMakeLists.txt

```
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(NONE)
+
+find_program(CDDL_GEN cddl-gen)
+if (NOT CDDL_GEN)
+  message(FATAL_ERROR
+    'cddl-gen was not found. Install it via pip. This script expects cddl-gen version 0.3.0')
+endif()
```

Cleaner to mark this `REQUIRED`:
```suggestion
find_program(CDDL_GEN cddl-gen REQUIRED)
```